lib/Cake/Test/Case/Cache/Engine/WincacheEngineTest.php

```diff
@@ -259,4 +259,27 @@ public function testGroupClear() {
 		$this->assertTrue(Cache::clearGroup('group_b', 'wincache_groups'));
 		$this->assertFalse(Cache::read('test_groups', 'wincache_groups'));
 	}
+
+/**
+ * Test that failed add write return false.
+ *
+ * @return void
+ */
+	public function testAdd() {
+
+		Cache::delete('test_add_key', 'wincache');
+
+		$expected = true;
+		$result = Cache::add('test_add_key', 'test data', 'wincache');
+		$this->assertEquals($expected, $result);
```

Comment:
You should use assertTrue() and assertFalse(). assertEquals() will not do type checks and the following will pass `assertEquals(true, 1);`.